client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -31,12 +32,12 @@ function ReaderSubscriptionListItem( {
 	followSource,
 	isEmailBlocked,
 } ) {
-	const siteTitle = getSiteName( { feed, site } );
+	const siteTitle = decodeEntities( getSiteName( { feed, site } ) );
```

Comment:
Is it worth having the helper methods (e.g. getSiteName) do `decodeEntities` instead?